examples/README.md

```diff
@@ -2,16 +2,17 @@
 
 This directory contains example YAML files to create and configure an Artemis broker on Kubernetes.
 
-If you are new to Artemis on Kubernetes, start with [a basic deployment](./artemis-basic-deployment.yaml):
-
+If you are new to Artemis on Kubernetes, start with [a basic deployment](artemis/artemis_single.yaml):
 
 1. Deploy the operator as described in the [Operator help](https://artemiscloud.io/docs/help/operator/#deploy-the-operator).
-
 2. Create a basic broker deployment:
 
 ```bash
-kubectl create -f examples/artemis-basic-deployment.yaml -n activemq-artemis-operator
+kubectl create -f examples/artemis/artemis_single.yaml -n <namespace>
 ```
 
 See https://artemiscloud.io/ for tutorials and more information about Artemis.
 
+For security enabled (TLS) connections example, please follow guide [Secured connection with ArtemisCloud Operator](https://artemiscloud.io/docs/tutorials/ssl_broker_setup).
+
+[ArtemisCloud Operator Security Configuration Example](https://github.com/artemiscloud/artemiscloud-examples/tree/main/operator/keycloak) provides example of JAAS configuration with Keycloak.
```

Comment:
@howardgao @brusdev @gtully can I link this item here? Is it ok?

Reply:
I would move this link to the doc